### PR TITLE
added SEQR + seqr-db

### PR DESCRIPTION
**Pull request by averagehat**
Adding seqr into the makefile.
Currently seqr requires java 8. 
`module load java/jdk1.8`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,6 +112,23 @@ $(DIAMOND): $(DIAMOND_TGZ)
...
+	wget $(SEQR_SRC_URL)
+	tar xvf seqr-clojure-$(SEQR_VER) 
+
+$(SEQR_DB): $(SEQR_SRC) $(SEQR_JAR) $(SMALLFASTA)
+	java -jar $(SEQR_JAR) index $(SMALLFASTA) --in_format fasta -d $(SEQR_SRC)/testdata/solr 
+
+# SEQR
+# *May* require java 8
+# Currently multi-cpu is not supported with embedded seqr because it causes database lock. it would work with server seqr.
+# Currently seqr is multi-threaded by default, not sure if it actually helps or not.
```
Comment (necrolyte2) [inline Makefile]: Does seqr support specifying how many threads?

Comment (averagehat) [inline Makefile]: No. And as far as I know It's not really taking advantage of the threads, when I look at the cpu usage

File: Makefile [hunk 2]
```diff
@@ -103,6 +112,23 @@ $(DIAMOND): $(DIAMOND_TGZ)
...
+single_cp_multi_thread_seqr.tsv: $(SEQR_DB) $(SEQR_JAR)
+	time java -jar $(SEQR_JAR) search $(BLASTQUERYFILE) $(SEQROPTIONS)
```
Comment (necrolyte2) [inline Makefile]: Does sqr dump to stdout? If so make sure to ``> $@``

**Comment by necrolyte2** [comment]
Add this to top so the report target will run on it as well.
You also don't seem to be creating the tsv file for sqr 
```
SQROUTPUT = single_cp_multi_thread_seqr.$(AVAILCPU).$(SUBSELECT).tsv
TSVOUTPUT = $(BLASTOUTPUT) $(DIAMONDOUTPUT) $(SQROUTPUT)
```

File: Makefile (hunk 1 above)
Comment (averagehat) [inline Makefile]: I need to do some benchmarking myself to find out how to actually benefit from multiple threads. I think the server version may benefit--but this is the embedded version.

**Comment by averagehat** [comment]
I got confused, I failed to pull your changes, should be good now

**Comment by necrolyte2** [comment]
Looks good to me

**Pull request merged**